hax/hax/hax.py

```diff
@@ -88,7 +88,7 @@ def main():
     # _run_qconsumer_thread function.
     #
     # [KN] Note: The server is launched in the main thread.
-    q = Queue(maxsize=8)
+    q = Queue(maxsize=32767)
```

Comment:
Looks like an overkill. I don't know what data structure is used under the hood of Queue, but it may happen that it's not an array but a linked list or something. So even maintenance of such a huge structure can bring additional performance penalties. That's why I'd recommend change sizes not that dramatically.

Why wouldn't it be 1K or 8K instead?

Reply:
Thanks!
Changed to 8191 as per discussion today

Reply:
Kindly squash the commits, just keep the latest one.